src/lib/ai/mental-llama/cli/test-prompts.ts

```
@@ -0,0 +1,494 @@
+  createCategoryTemplate,
+  buildPrompt,
+  MentalHealthCategory,
+  addEmotionalContext,
+  applyChainOfThought,
+  createSelfConsistencyPrompts
+} from '../prompts';
+import {
+  evaluatePrompt,
+  createComprehensiveTestDataset,
+  PromptEvaluationOptions
+} from '../evaluator';
```

ESLint

[error] Imports "PromptEvaluationOptions" are only used as type.

(@typescript-eslint/consistent-type-imports)